patches/third_party-blink-renderer-bindings-core-v8-referrer_script_info.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/third_party/blink/renderer/bindings/core/v8/referrer_script_info.h b/third_party/blink/renderer/bindings/core/v8/referrer_script_info.h
+index a488b931fa0bd9a3e57376781ab5049fbc59d4cd..d595b9938ebe9b39f1a0311352a43f4472745f4f 100644
+--- a/third_party/blink/renderer/bindings/core/v8/referrer_script_info.h
++++ b/third_party/blink/renderer/bindings/core/v8/referrer_script_info.h
+@@ -39,7 +39,7 @@ class CORE_EXPORT ReferrerScriptInfo {
+                            options.CredentialsMode(),
+                            options.Nonce(),
+                            options.ParserState(),
+-                           options.GetReferrerPolicy()) {}
```

Comment:
```
#define ReferrerScriptInfo ReferrerScriptInfo_ChromiumImpl
class CORE_EXPORT ReferrerScriptInfo : ReferrerScriptInfo_ChromiumImpl ...
``` ?

Reply:
there's `static ReferrerScriptInfo ReferrerScriptInfo::FromV8HostDefinedOptions()` that I need to override.

with this define it will become
`ReferrerScriptInfo_ChromiumImpl ReferrerScriptInfo_ChromiumImpl::FromV8HostDefinedOptions()` which won't work as I must return `ReferrerScriptInfo` instead:
```
ReferrerScriptInfo ReferrerScriptInfo::FromV8HostDefinedOptions(...) {
  auto info = ReferrerScriptInfo_ChromiumImpl::FromV8HostDefinedOptions(...);
  // need to fill it with my values, but I can't.
  return info; // this will fail as it's not ReferrerScriptInfo;
}
```

Reply:
it would be `auto info = ReferrerScriptInfo(ReferrerScriptInfo_ChromiumImpl::FromV8HostDefinedOptions(...));` where you'd copy the values from ReferrerScriptInfo_ChromiumImpl to ReferrerScriptInfo

Reply:
> it would be `auto info = ReferrerScriptInfo(ReferrerScriptInfo_ChromiumImpl::FromV8HostDefinedOptions(...));` where you'd copy the values from ReferrerScriptInfo_ChromiumImpl to ReferrerScriptInfo in the ReferrerScriptInfo constructor

this doesn't look very good. It's a runtime overhead even with a disabled feature. Do you really want this?